architecture/core_concepts/routes.adoc

```diff
@@ -186,21 +186,46 @@ addresses; because of the NAT configuration, the originating IP address
 
 *Configuration Parameters*
 
-With all the items outlined in this section, you should be able to set environment
+For all the items outlined in this section, you can set environment
 variables on the *deployment config* for the router to alter its configuration.
 
 .Router Configuration Parameters
-[cols="2*", options="header"]
+[cols="3*", options="header"]
 |===
-|Variable | Default
-|`*ROUTER_SYSLOG_ADDRESS*` | ""
-|`*ROUTER_LOG_LEVEL*` | warning
-|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms
-|`*ROUTER_DEFAULT_CONNECT_TIMEOUT*`| 5s
-|`*ROUTER_DEFAULT_CLIENT_TIMEOUT*`| 30s
-|`*ROUTER_DEFAULT_SERVER_TIMEOUT*`| 30s
-|`*ROUTER_DEFAULT_TUNNEL_TIMEOUT*` | 1h
-|`*ROUTER_SLOWLORIS_TIMEOUT*` | 10s
+|Variable | Default | Description
+|`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
+|`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named tls.crt. If tls.crt is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if DEFAULT_CERTIFICATE or DEFAULT_CERTIFICATE_PATH are not specified.
+|`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if DEFAULT_CERTIFICATE is not specified.
+|`*EXTENDED_VALIDATION*` | true | Whether to perform an additional extended validation step on all routes admitted by this router.
+|`*NAMESPACE_LABELS*` |  | A label selector to apply to namespaces to watch, empty means all.
+|`*PROJECT_LABELS*` |  | A label selector to apply to projects to watch, emtpy means all.
+|`*RELOAD_SCRIPT*` |  | The path to the reload script to use to reload the router.
+|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | How frequently to check backends for "liveness".
+|`*ROUTER_COMPRESSION_MIME*` | "text/html text/plain text/css" | A space separated list of mime types to compress.
+|`*ROUTER_DEFAULT_CLIENT_TIMEOUT*`| 30s | How long a client has to acknowledge or send data.
+|`*ROUTER_DEFAULT_CONNECT_TIMEOUT*`| 5s | The maximum connect time.
+|`*ROUTER_DEFAULT_SERVER_TIMEOUT*`| 30s | How long a server has to acknowledge or send data.
+|`*ROUTER_DEFAULT_TUNNEL_TIMEOUT*` | 1h | How long to allow TCP or WebSocket connections to remain open.
+|`*ROUTER_ENABLE_COMPRESSION*`| false | If "true", compress responses when possible.
+|`*ROUTER_LOG_LEVEL*` | warning | What log level to send to the above address.
+|`*ROUTER_OVERRIDE_HOSTNAME*`|  | If set, override the spec.host value for a route with the template in ROUTER_SUBDOMAIN.
+|`*ROUTER_SERVICE_HTTPS_PORT*` | 443 | What port to listen for https requests on.
```

Comment:
Suggestion: Please change "What port to listen for https requests on." to "Port to listen for https requests." OR "Port on which to listen for https requests."